app/src/main/java/com/xmartlabs/taskloans/data/model/service/AuthResponse.kt

```diff
@@ -6,7 +6,7 @@ import com.xmartlabs.taskloans.data.model.User
 /**
  * Created by mirland on 03/05/20.
  */
-data class SignInResponse(
+data class AuthResponse(
     @SerializedName("auth_token")
     val token: String?,
```

Comment:
maybe non nullable? the token comes always with the respose  isn't it?

Reply:
Yes, I agree but we talked with mirland and he suggested that change.